run-build-functions.sh

```diff
@@ -296,13 +300,73 @@ install_dependencies() {
     exit 1
   fi
 
+  # remove the v in front of the node version
+  local cleanNodeVersion=$(echo "$NODE_VERSION"| sed "s/[^[:digit:].-]//g")
+  # Node.js Corepack
+  # corepack was packported to 14.19.0
+  dpkg --compare-versions "$cleanNodeVersion" "ge" "14.19.0"
+
+  if [ $? -eq 0 ]; then
+    # corepack was added in 16.9.0 and packported to 14.19.0
+    dpkg --compare-versions "$cleanNodeVersion" "ge" "16.9.0"
+    if [ $? -eq 0 ] || [ "$(get_major_version $cleanNodeVersion)" == "14" ]; then
+      echo "Enabling node corepack"
+      corepack enable
+    fi
+  fi
```

Comment:
Would it be possible to just check via `which corepack` or run a `corepack` command and via its error code detect if corepack is present or not and log it accordingly? Thinking it would be easier and would avoid the version comparison and detection/

Reply:
sure! Good point